includes/class-munim.php

```diff
@@ -142,18 +142,18 @@ public function register_assets() {
 
 		wp_register_script(
 			'munim-script',
-			MUNIM_PLUGIN_URL . 'assets/js/script.js',
+			MUNIM_PLUGIN_URL . 'assets/js/script.min.js',
 			[ 'apexcharts-script', 'munim-clipboard', 'jquery' ],
-			filemtime( MUNIM_PLUGIN_DIR . 'assets/js/script.js' ),
+			filemtime( MUNIM_PLUGIN_DIR . 'assets/js/script.min.js' ),
 			true
 		);
 		wp_enqueue_script( 'munim-script' );
 
 		wp_register_script(
 			'munim-dashboard',
-			MUNIM_PLUGIN_URL . 'assets/js/dashboard.js',
+			MUNIM_PLUGIN_URL . 'assets/js/dashboard.min.js',
 			[],
-			filemtime( MUNIM_PLUGIN_DIR . 'assets/js/dashboard.js' ),
+			filemtime( MUNIM_PLUGIN_DIR . 'assets/js/dashboard.min.js' ),
 			true
 		);
 		wp_enqueue_script( 'munim-dashboard' );
```

Comment:
Currently, min assets are loaded alway.

We should load them only when `WP_DEBUG` is set to false else load unminified assets.

```
// Use minified libraries if SCRIPT_DEBUG is turned off
$suffix = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';


wp_enqueue_script( 'my-script-handle', plugin_dir_url( __FILE__ ) . 'assets/my-file' . $suffix . '.js', array( 'jquery' ) );
```

Reply:
Okay.

Reply:
I found` $suffix = ( defined( 'WP_DEBUG' ) && true === WP_DEBUG ) ? '' : '.min';` working!

Reply:
it should be on `SCRIPT_DEBUG` not on `WP_DEBUG` the way core handles internally

Reply:
Oh! Okay I understand.